src/main/java/org/synyx/urlaubsverwaltung/overtime/Overtime.java

```diff
@@ -46,17 +48,17 @@ protected Overtime() {
         // OK
     }
 
-    public Overtime(Person person, LocalDate startDate, LocalDate endDate, BigDecimal numberOfHours) {
+    public Overtime(Person person, LocalDate startDate, LocalDate endDate, Duration duration) {
 
         Assert.notNull(person, "Person must be given.");
```

Comment:
Würde ich nun nicht in diesem PR machen, aber mal so aus interesse. Meinst du wir sollten in der UV weiterhin die Asserts drin lassen und eher auf eine defensitve Programmierung gehen oder rausnehmen und erwarten, das hier kein null übergeben wird?

Reply:
Ja klar, sehe das genauso, sobald die Validierung aus dem Frontend durch ist, sollte hier kein `null` mehr möglich sein. Ist hier nur zur Konsistenz des bisherigen Codes auch für die `duration` drin. Finde die Idee das in einem eigenen PR zu machen gut.

Reply:
Ich hatte das tatsächlich mal alles rausgenommen, war mir dann aber unsicher wie wir im Team dazu stehen. Zudem wurde ab und an auch Programmatisch auf die Asserts reagiert soweit ich mich noch dran erinnere. Eigener PR ist gut.